engine/src/main/java/org/destinationsol/mercenary/MercenaryUtils.java

```diff
@@ -0,0 +1,77 @@
+package org.destinationsol.mercenary;
+
+import org.destinationsol.Const;
+import org.destinationsol.common.SolMath;
+import org.destinationsol.game.Faction;
+import org.destinationsol.game.ShipConfig;
+import org.destinationsol.game.SolGame;
+import org.destinationsol.game.input.AiPilot;
+import org.destinationsol.game.input.Guardian;
+import org.destinationsol.game.item.MercItem;
+import org.destinationsol.game.planet.Planet;
+import org.destinationsol.game.ship.SolShip;
+import org.destinationsol.game.ship.hulls.HullConfig;
+
+import com.badlogic.gdx.math.Vector2;
+
+public class MercenaryUtils {
+
+    /**
+     * Creates a new mercenary, SolShip and all. It also adds the MercItem and SolShip to the
+     * proper registries and sets the proper variables.
+     * @param game The instance of the game we're dealing with
+     * @param hero The player
+     * @param mercItem The mercenary item to build everything else off of
+     * @return True if the ship could be created, false if not
+     */
+    public static boolean createMerc(SolGame game, SolShip hero, MercItem mercItem) {
+        ShipConfig config = mercItem.getConfig();
+        Guardian dp = new Guardian(game, config.hull, hero.getPilot(), hero.getPosition(), hero.getHull().config, SolMath.rnd(180));
+        AiPilot pilot = new AiPilot(dp, true, Faction.LAANI, false, "Merc", Const.AI_DET_DIST);
+        Vector2 pos = getPos(game, hero, config.hull);
+        if (pos == null) {
+            return false;
+        }
+        SolShip merc = game.getShipBuilder().buildNewFar(game, pos, new Vector2(), 0, 0, pilot, config.items, config.hull, null, true, config.money, null, true)
+                .toObj(game);
+        
+        merc.setMerc(mercItem);
+        mercItem.setSolShip(merc);
+        
+        game.getHero().getTradeContainer().getMercs().add(mercItem);
+        game.getObjMan().addObjNow(game, merc);
+        return true;
+    }
+    
+    /**
+     * 
+     * @param game The instance of the game we're dealing with
+     * @param hero The player
+     * @param hull The hull of the mercenary in question
+     * @return The position to spawn the mercenary at. Null for no available position.
```

Comment:
`The position to spawn the mercenary at. Null for no available position.` -> `The position to spawn the mercenary at, or null for no available position.`